server/jetstream.go

```diff
@@ -146,7 +146,10 @@ const (
 	JetStreamUpdateMsgWithRevision = "$JS.%s.UPDATE"
 
 	// JetStreamObservableAckSamplePre is the prefix for sample messages from observables
-	JetStreamObservableAckSamplePre = "$JS.OBSERVABLE.ACKSAMPLE"
+	JetStreamObservableAckSamplePre = "$JS.EVENTS.OBSERVABLE.ACKSAMPLE"
```

Comment:
Would do $JS.EV... Just b/c will be duplicated so much.

Also if these are pre then we need to make a decision on naming for streams and consumers, whether or not they can be multi-token and how we deal with events where we want both the stream and observable.

For streams we allow (for now) with templates. We name the auto-created stream with the literal subject name that triggered its creation.

We could make the canonical with simple mapping.

My.Stream.Derek => My_Stream_Derek? 

This would still allow a priori knowledge in clients etc but preserve our ability to do authorization via subjects.